src/layers/VolumeLayer/VolumeLayer.js

```diff
@@ -88,12 +92,16 @@ const VolumeLayer = class extends CompositeLayer {
         progress += 0.5 / totalRequests;
         this.setState({ progress });
       };
+      const abortController = new AbortController();
+      this.setState({ abortController });
+      const { signal } = abortController;
```

Comment:
Added this is `ImageLayer` to improve performance when changing z-slice quickly so figured I'd do the same here.